frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.tsx

```diff
@@ -52,7 +52,7 @@ const RevisionsOverviewList: React.FC<RevisionsOverviewListProps> = ({ revisions
         <Button
           variant="secondry"
           onClick={() => trafficModalLauncher({ obj: service })}
-          disabled={revisions.length === 0}
+          disabled={!revisions || revisions.length === 0}
```

Comment:
can we use `_.isEmpty(revisions )` or `!(revisions && revisions.length)`

Reply:
> nit : can we use `_.isEmpty(revisions )` or `!(revisions && revisions.length)`
@invincibleJai both works fine.

Reply:
@sahil143  ^^

Reply:
fixed in https://github.com/openshift/console/pull/3205/commits/ec08d5e75df65d8cbe0ff7a8571dc5d1c7a826cf

Reply:
@sahil143 doesn't like lodash ;)

Reply:
Yeah :P